packages/forms/src/UIForm/Message/Message.component.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+
+export default function Message(props) {
+	const {
+		errorMessage,
+		description,
+		isValid,
+	} = props;
+
+	const message = isValid ? description : errorMessage;
+	return message ?
+		(
+			<div
+				className="help-block"
```

Comment:
In fact I'm using [Bootstrap Form Control Validation classes](http://getbootstrap.com/css/#forms-control-validation)